cypress/integration/invitesMembers.js

```diff
@@ -95,7 +95,6 @@ describe('Invite Signup', () => {
         cy.get('[data-attr=change-membership-level]').last().should('contain', 'member')
         cy.get('[data-attr=change-membership-level]').last().click()
         cy.get('[data-test-level=8]').click()
-        cy.get('[data-attr=change-membership-level]').last().should('contain', 'administrator')
```

Comment:
Should this change be in a separate PR? It's (I guess) not related to this change...

Reply:
Actually this was a flaky test, that's why I included it here, sorry forgot the context